extensions/amp-story/0.1/system-layer.js

```diff
@@ -220,6 +227,14 @@ export class SystemLayer {
    * @param {!Event} e
    * @private
    */
+  onEscapeKey(e) {
+    this.dispatch_(EventType.CLOSE_BOOKEND, e);
```

Comment:
This is slightly odd, since the bookend isn't always open.  It will trigger the event even if the bookend isn't open, which will just be a no-op, but I wonder if that's okay?